app/src/types/SubmissionSearchParameters.ts

```diff
@@ -0,0 +1,3 @@
+export type SubmissionSearchParameters = {
+  submissionId?: string[];
```

Comment:
Are we standardizing on Generic notation `Array<string>` or bracket notation? Need to make sure we're consistent.